libs/vkd3d/device.c

```diff
@@ -2836,6 +2836,25 @@ static HRESULT STDMETHODCALLTYPE d3d12_device_CheckFeatureSupport(d3d12_device_i
             return S_OK;
         }
 
+        case D3D12_FEATURE_SHADER_CACHE:
+        {
+            D3D12_FEATURE_DATA_SHADER_CACHE *data = feature_data;
+
+            if (feature_data_size != sizeof(*data))
+            {
+                WARN("Invalid size %u.\n", feature_data_size);
+                return E_INVALIDARG;
+            }
+
+            /* FIXME MSDN says SINGLE_PSO is always supported, but we
+             * currently don't support the API, including GetCachedBlob. */
+            WARN("Shader cache features not supported.");
```

Comment:
I think we can only support this when we have more dynamic state to play around with. App might ask for cached pipeline before we have actually compiled anything meaningful.